src/Symfony/Component/PropertyInfo/Extractor/ReflectionExtractor.php

```diff
@@ -426,19 +454,23 @@ public function getWriteInfo(string $class, string $property, array $context = [
             }
         }
 
-        [$adderAccessName, $removerAccessName, $adderAndRemoverErrors] = $this->findAdderAndRemover($reflClass, $singulars);
-        if ($allowAdderRemover && null !== $adderAccessName && null !== $removerAccessName) {
-            $adderMethod = $reflClass->getMethod($adderAccessName);
-            $removerMethod = $reflClass->getMethod($removerAccessName);
+        $camelized = $this->camelize($property);
 
-            $mutator = new PropertyWriteInfo(PropertyWriteInfo::TYPE_ADDER_AND_REMOVER);
-            $mutator->setAdderInfo(new PropertyWriteInfo(PropertyWriteInfo::TYPE_METHOD, $adderAccessName, $this->getWriteVisibilityForMethod($adderMethod), $adderMethod->isStatic()));
-            $mutator->setRemoverInfo(new PropertyWriteInfo(PropertyWriteInfo::TYPE_METHOD, $removerAccessName, $this->getWriteVisibilityForMethod($removerMethod), $removerMethod->isStatic()));
+        if (null === $adderAccessName || null === $removerAccessName) {
```

Comment:
The `$adderAccessName` and `$removerAccessName` have been defined in the metadata.